project/CommonSettings.scala

```diff
@@ -49,4 +49,6 @@ object CommonSettings {
   lazy val configure: Project => Project = (project: Project) => {
     project.settings(CommonSettings.commonSettings: _*)
   }
+
+  val dottyVersion = "0.24.0-RC1"
```

Comment:
I would remove this as it's more convenient to set the Scala version in `project/Dependencies`

Reply:
So `Versions` or `Dependencies`?

Reply:
I made a note on that in the overall review comment.